binr/rahash2/rahash2.c

```diff
@@ -321,9 +319,9 @@ static void algolist() {
 	eprintf ("\n");
 	eprintf ("Available Encoders/Decoders: \n");
 	// TODO: do not hardcode
-	eprintf ("  base64\n");
-	eprintf ("  base91\n");
-	eprintf ("  punycode\n");
+	printf ("  base64\n");
+	printf ("  base91\n");
+	printf ("  punycode\n");
 	eprintf ("\n");
 	eprintf ("Available Crypto Algos: \n");
```

Comment:
well, those were already there like that.
i just changed them to printf since it was following the others printfs